modules/virt-reclaiming-statically-provisioned-persistent-volumes.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reusing-persistent-volumes.adoc
+
+[id="virt-reclaiming-statically-provisioned-persistent-volumes_{context}"]
+= Reclaiming statically provisioned persistent volumes 
+
+Reclaiming a statically provisioned persistent volume (PV) for re-use is dependent on the underlying storage.
+This procedure provides a general approach that might need to be customized depending on your storage.
+
+.Procedure
+
+. Ensure that the reclaim policy of the PV is set to `Retain`:
++
+----
+$ oc get pv <pv_name> -o yaml | grep 'persistentVolumeReclaimPolicy'
+----
+
+.. If the `persistentVolumeReclaimPolicy` is not set to `Retain`, edit the reclaim policy with the following command:
++
+----
+$ oc patch pv <pv_name> -p '{"spec":{"persistentVolumeReclaimPolicy":"Retain"}}'
+----
+
+. Ensure that no resources are using the persistent volume claim (PVC):
++
+----
+$ oc describe pvc <pvc_name> | grep 'Mounted By:'
+----
++
+Remove any resources that are still using the PVC before continuing.
+
+. Delete the PVC to release the PV:
++
+----
+$ oc delete pvc <pvc_name>
+----
+
+. Optional: Export the PV configuration to a YAML file. If you need to manually remove the shared storage later in this procedure, you can refer to this configuration.
+You can also use `spec` parameters in this file as the basis to create a new PV with the same storage configuration after you have reclaimed the PV:
++
+----
+$ oc get pv <pv_name> -o yaml > <file_name>.yaml
+----
+
+. Delete the PV:
++
+----
+$ oc delete pv <pv_name>
+----
+
+. Optional: Depending on the storage type, you may need to remove the contents of the shared storage folder:
++
+----
+$ rm -rf <path_to_share_storage>
+----
+
+. Optional: You can now create a PV that uses the same configuration as the deleted PV:
++
+[NOTE]
+====
+To avoid possible conflict, it is good practice to give the new PV a different name to the one that was deleted.
+====
++
+----
+$ oc create -f <new_pv_name>.yaml
```

Comment:
The file name itself is not important, but the name of the new PV object itself should be different (it's not mandatory, but it's recommended as it miiight introduce an error). We can't use the same file that we exported earlier because it's filled with all kinds of metadata - when I create a PV using it openshift basically just returns the dead PV to the cluster. 

with that in mind I've clarified this step and the note:
. Optional: Create a PV that uses the same storage configuration as the deleted PV. If you exported the reclaimed PV configuration earlier, you can use the `spec` parameters of that file as the basis for a new PV manifest:
+
[NOTE]
====
To avoid possible conflict, it is good practice to give the new PV object a different name than the one that you deleted.
====